### PR TITLE
Add script to push release tag

### PR DESCRIPTION
**Pull request by christian-bromann**
## Proposed changes

It seems that Lerna doesn't publish annotated tags to GitHub
after the release. This script is a little helper to ensure
this happens.

## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [ ] I have added proper type definitions for new commands (if appropriate)

## Further comments

This is necessary to ensure that if someone releases WebdriverIO someone else picks up the release tag so that the next release only includes packages where changes were made. Lerna currently uses `git describe` for that and it appears that it only pushes lightweight tags during the release. Not sure if there is an option to set to change that but this is the best workaround I found so far.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,8 +21,9 @@
     "clean-full:package_modules": "rimraf ./packages/*/node_modules",
     "generate:typings": "node ./scripts/type-generation/generate-all-typings.js",
     "link": "node ./scripts/link",
-    "release": "NODE_ENV=production npm run setup-full && npm run generate:typings && lerna publish --exact",
+    "release": "NODE_ENV=production npm run setup-full && npm run generate:typings && lerna publish --exact && npm run pushTags",
     "release:next": "npm run release -- --dist-tag next",
+    "pushTags": "./scripts/pushTags.js",
```
Comment (emilyrohrbough) [inline package.json:26]: Could you configure this as the postPublish script?

Comment (christian-bromann) [reply package.json:26]: good idea 👍

**Review by emilyrohrbough — APPROVED** [review]

**Review by mgrybyk — COMMENTED** [review]
Let's try the proposal for the next release.
I guess we can do it manually before merge.

Is it safe to do a force push to master?

**Comment by christian-bromann** [comment]
@mgrybyk I changed the command to `git push origin refs/tags/v${version} -f --no-verify` to make sure we only push the tag. I will try this on the next release.

**Comment by christian-bromann** [comment]
The solution seems to work. We can't use the `postPublish` hook though because the root `package.json` is never published to NPM. The result of the script shows that only the tag was pushed:

```js
> ./scripts/pushTags.js


Pushing release tag...
To github.com:webdriverio/webdriverio.git
 + 2084b3784...26289a9c7 v6.4.7 -> v6.4.7 (forced update)
```

I checked out the repo fresh after the release and it showed the latest tag via `git describe`.

**Pull request merged**